controllers/apply-next/simple/fields.js

```diff
@@ -184,14 +185,37 @@ const allFields = {
             `,
             cy: 'TODO'
         },
-        type: 'textarea',
+        type: 'budget',
         attributes: {
-            rows: 12
+            max: MAX_BUDGET_TOTAL,
+            rowLimit: 10
         },
         isRequired: true,
-        schema: Joi.string().required(),
+        schema: commonValidators.budgetField(MAX_BUDGET_TOTAL),
         messages: {
-            base: { en: 'Enter a project budget', cy: '' }
+            base: { en: 'Enter a project budget', cy: '' },
+            'any.empty': { en: 'Please supply both an item name and a cost', cy: '' },
+            'number.base': { en: 'Make sure each cost is a valid number', cy: '' },
+            'budgetItems.overBudget': {
+                en: `Ensure your budget total is £${MAX_BUDGET_TOTAL.toLocaleString()} or less.`,
+                cy: `(WELSH) Ensure your budget total is £${MAX_BUDGET_TOTAL.toLocaleString()} or less.`
+            }
+        },
+        displayFormat(value) {
+            if (!isArray(value)) {
+                return value;
+            } else {
+                const total = value.reduce((acc, cur) => {
```

Comment:
Not sure how I feel about this whole thing. Html here feels unpleasant, but I'm not sure I have a better idea at the moment.

Reply:
Only option I can think of is to pass through the path of a template fragment to use when rendering it but that feels like overkill. Otherwise this code could move to the summary screen directly but that sort of defeats the object of this function.

Reply:
Yeah I think it's fine. Perhaps try a template string to avoid the string concatenating, otherwise probably makes sense.

Reply:
Just tried that and the ` | nl2br` filter on the summary screen clobbers it (adds in extra markup etc):

![image](https://user-images.githubusercontent.com/394376/54039047-e0560600-41b9-11e9-9120-41d9c921706d.png)

Will leave as-is for now.

Reply:
👍